setup.py

```diff
@@ -21,16 +21,11 @@
     author="nielstron",
     author_email="n.muendler@web.de",
     url="https://github.com/nielstron/pysyncthru/",
-    py_modules=["pysyncthru"],
-    packages=find_packages(),
+    packages=find_packages(exclude=("pysyncthru.tests", "pysyncthru.tests.*")),

```

Comment:
Actually, would "pysyncthru.tests*" not have sufficed for excluding all tests?

Reply:
It would have, yes. Ditto `*.tests*` or even `*tests*`. Mostly a matter of taste, it boils down to how accurately one wants to pinpoint the exclusions, e.g. in order to avoid inadvertently excluding something necessary. Those more relaxed globs are rarely an issue though, but as a hypothetical/imaginary example, `*tests*` would match the `some.package.lateststate` package.